setup.py

```diff
@@ -12,6 +12,7 @@
         "eth-bloom>=1.0.0,<2.0.0",
         "eth-utils>=1.0.1,<2.0.0",
         "eth-typing>=1.0.0,<2.0.0",
+        "juno>=0.1.0",
```

Comment:
I'm confused, https://pypi.org/project/juno/ seems to be something different, no? Where is that dependency coming from?

Reply:
Ah, ok I see it is here: https://github.com/lrettig/juno But then, how does that fit in with that `juno` package on pypi that seems to be occupied by something else, no?

Reply:
It doesn't. Two different things, no connection. The name can be changed :) I'm just running this locally, no pypi.